webapp/src/components/ParcelCard/ParcelCard.css

```diff
@@ -2,11 +2,18 @@
   width: calc(100% - 14px) !important;
 }
 
-.ParcelCard .mortgage-status.open{
+.ParcelCard .mortgage-status{
   text-transform: capitalize;
+}
+
+.ParcelCard .Mortgage.pending{
   color: var(--orange);
 }
 
+.ParcelCard .mortgage-status.ongoing{
+  color: var(--light-blue);
+}
```

Comment:
The annoying co-worker strikes again. Can we prettify this file? to look like `.selector {` instead of `.selector{`

Reply:
Same goes for the other `.css` files, but then again, it's not really important